backend/src/auth_service/connection_package/persistence.rs

```diff
@@ -124,7 +124,7 @@ pub(crate) mod tests {
         ConnectionPackage::new_for_test(
             ConnectionPackagePayload {
                 verifying_key,
-                protocol_version: MlsInfraVersion::default(),
+                protocol_version: AirProtocolVersion::default(),
```

Comment:
Since we are touching this place, maybe we should explicitly use the protocol version here?